view/frontend/templates/category_merchandising.phtml

```diff
@@ -42,7 +42,8 @@
         "*": {
             "Nosto_Cmp/js/cmp": {
                 "merchant": "<?php echo $block->getNostoAccount(); ?>",
-                "cookieName": "<?php echo $block->getCookieName(); ?>",
+                "categoryCookie": "<?php echo $block->getCategoryMappingCookieName(); ?>",
```

Comment:
Shall we move to short syntax and escaping like we are doing in Magento?
https://github.com/Nosto/nosto-magento2/blob/097a7d0af7a7ba8d0a34a20f75c6eea6621b3ea7/view/adminhtml/templates/iframe.phtml#L49